### PR TITLE
Add safe-logging as a dependency of atlasdb-cassandra and timelock-impl

### PR DESCRIPTION
**Pull request by fsamuel-bs**
**Goals (and why)**: Fix develop.

**Implementation Description (bullets)**: -

**Concerns (what feedback would you like?)**: -

**Where should we start reviewing?**: -

**Priority (whenever / two weeks / yesterday)**: -

---
This change is [Reviewable](https://reviewable.io/reviews/palantir/atlasdb/2171)

### CODE REVIEW

**Comment by hsaraogi** [comment]
```
13:43 $ ag safeArg | grep java | cut -d ':' -f 1 | cut -d '/' -f 1 | uniq -c
  32 atlasdb-cassandra
   2 atlasdb-config
  10 atlasdb-impl-shared
   9 lock-impl
   3 timelock-impl
   2 timestamp-impl
```

**Comment by hsaraogi** [comment]
Can you add safe arg to all the above projects please, if they are not already added?

**Comment by fsamuel-bs** [comment]
Added!

File: lock-impl/build.gradle [hunk 1]
```diff
@@ -8,8 +8,11 @@ apply from: "../gradle/shared.gradle"
 dependencies {
   compile project(":lock-api")
   compile project(":atlasdb-commons")
+
   compile 'com.palantir.patches.sourceforge:trove3:' + libVersions.trove
   compile group: 'com.palantir.remoting2', name: 'tracing'
+  compile group: 'com.palantir.safe-logging', name: 'safe-logging'
+
```
Comment (hsaraogi) [inline lock-impl/build.gradle]: nit: awkward new line..

**Pull request merged**